src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -18,10 +18,10 @@ package com.squareup.kotlinpoet
 import com.google.common.collect.ImmutableMap
 import com.google.common.truth.Truth.assertThat
 import com.google.testing.compile.CompilationRule
-import com.squareup.kotlinpoet.KModifier.ABSTRACT
 import com.squareup.kotlinpoet.KModifier.INTERNAL
-import com.squareup.kotlinpoet.KModifier.PRIVATE
+import com.squareup.kotlinpoet.KModifier.ABSTRACT
```

Comment:
Seems like we lost a bit of alphabetization here... ABSTRACT should be up where it used to be.